src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -424,6 +424,30 @@ make_coordinate_map_base(Maps&&... maps) noexcept {
       std::forward<Maps>(maps)...);
 }
 
+/// \ingroup ComputationalDomainGroup
+/// \brief Creates a std::vector<std::unique_ptr<CoordinateMapBase>
+/// containing the result of make_coordinate_map_base applied to each
```

Comment:
`make_coordinate_map_base` (ticks)

I wonder if something like:

Creates a vector of pointers to `CoordinateMapBase` of the concrete maps passed in in the order they are passed in.

\requires `not flat_all<not tt::is_a_v<CoordinateMap, Arg1>, (not tt::is_a_v<CoordinateMap, Args>)...>`


I don't think we ever want `CoordinateMap`s passed here, I think. Right?

Reply:
Since `arg_1` and `remaining_args` are just forwarded to make_coordinate_map base, this should work as expected if a `CoordinateMap` is passed in and just copy or move construct it I think. I can't see a reason why we would want to disallow passing `CoordinateMap`'s in.

Reply:
You're probably right. We should probably have a test case for this just to be sure, though. I also don't really see a reason to prohibit passing in a `CoordinateMap`